### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1379] [Feedback]
Use this place to discuss the [error component](https://ons-design-system.netlify.com/components/error) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by bameyrick** [comment]
This component is used by the [errors pattern](#30).

**Comment by bameyrick** [comment]
Errors don't work when using Jinja as the render engine due to caller always scopes to the closest macro. See: https://github.com/pallets/jinja/issues/371

**Comment by jrbarnes9** [comment]
Markdown table not rendering in '[Error component](https://ons-design-system.netlify.com/components/error/)' documentation...
<img width="649" alt="Screenshot 2019-11-29 at 15 00 59" src="https://user-images.githubusercontent.com/43346934/69877103-28253c00-12b9-11ea-9ce4-aba50a46c530.png">

**Comment by jrbarnes9** [comment]
We should show an example of the error component or at least link to the pattern
